libc/src/__support/FPUtil/CMakeLists.txt

```diff
@@ -82,8 +82,11 @@ add_header_library(
     BasicOperations.h
   DEPENDS
     .fp_bits
+    .fenv_impl
     libc.src.__support.CPP.type_traits
+    libc.src.__support.uint128
     libc.src.__support.common
+    libc.src.__support.macros.optimization.h
```

Comment:
I was curious why your tests were not running in the buildkite pre-submit check.  Turn out I missed this one:
it should be `libc.src.__support.macros.optimization`, without `.h`.  After updating this dependency, you should be able to build and see some of the issues with the tests when running `ninja libc`.

Reply:
Ah so that was it. I was wondering the same thing since morning when I took a look at the buildkite pre-submit check. Thanks for pointing it out.